plaid-stl/src/github/mod.rs

```diff
@@ -843,12 +936,173 @@ pub fn configure_secret(
     let request =
         serde_json::to_string(&params).map_err(|_| PlaidFunctionError::ErrorCouldNotSerialize)?;
 
-    let res = unsafe {
-        github_configure_secret(request.as_bytes().as_ptr(), request.as_bytes().len())
-    };
+    let res =
+        unsafe { github_configure_secret(request.as_bytes().as_ptr(), request.as_bytes().len()) };
 
     match res {
         0 => Ok(()),
         x => Err(x.into()),
     }
 }
+
+/// Search for files with given filename in a GitHub organization's code.
+/// If additional selection criteria are given, these are used to decide whether
+/// results are selected or discarded.
+///
+/// **Arguments:**
+/// - `filename`: The name of the files to search, e.g., "README.md"
+/// - `org`: The name of the GitHub organization
+/// - `selection_criteria`: An optional `FileSearchSelectionCriteria` object with additional criteria
+pub fn search_file(
```

Comment:
Nit. 
search_file => implies searching within a file
search_for_file => implies looking for a file